include/Basis/transform.hpp

```diff
@@ -98,4 +99,61 @@ void basisRotate(SingleParticleBasis<double, ShellType>& sp_basis, SQHamiltonian
 }
 
 
+/**
+ * Unrestricted
```

Comment:
Immediately after merging this PR, could you open an issue to implement `USpinorBasis`?